charts/victoria-metrics-distributed/values.yaml

```diff
@@ -0,0 +1,222 @@
+# -- overrides the chart's name
+nameOverride: "vm-distributed"
+
+# -- overrides the chart's computed fullname.
+fullnameOverride: ""
+
+# -- set up a vmauth as the global write entrypoint
+vmauthIngestGlobal:
+  enabled: true
+  name: ""
+  spec: {}
+
+# -- set up a vmauth as the global read entrypoint
+vmauthQueryGlobal:
+  enabled: true
+  name: ""
+  spec: {}
+
+# -- config per availability zone components, including vmagent, vmcluster, vmauth etc
+availabilityZones:
+  - name: zone-eu-1
+    # -- allow data ingestion to this zone
+    allowIngest: true
+    # -- allow data query from this zone through global query endpoint
+    allowQuery: true
+    # -- nodeselector to restrict where pods of this zone can be placed.
+    # usually provided by cloud providers.
+    nodeSelector:
+      topology.kubernetes.io/zone: zone-eu-1
```

Comment:
The scenario of using `nodeAffinity` instead of `nodeSelector` for zone assignment is not taken into account.

Reply:
added in https://github.com/VictoriaMetrics/helm-charts/pull/960/commits/4e3204ed23a119eb48b9a2eb6d9a8ce8f84156a8, please check.